dataprofiler/profilers/profile_builder.py

```diff
@@ -444,6 +445,361 @@ def clean_data_and_get_base_stats(df_series, sample_size,
+                             "ProfileOptions object.")
+
+        self._profile = None
+        self.options = options
+        self.encoding = None
+        self.file_type = None
+        self._min_true_samples = min_true_samples
+        self._samples_per_update = samples_per_update
+
+        # base stats
+        self.total_samples = 0
+        self._empty_line_count = 0
+
+        # TODO: allow set via options
+        self._sampling_ratio = 0.2
+        self._min_sample_size = 5000
+
+        # assign data labeler
+        # TODO: when available, update to use "unstructured_options"
+        self.options.structured_options.unstructured_text = TextProfilerOptions()
+        data_labeler_options = self.options.structured_options.data_labeler
+        if data_labeler_options.is_enabled \
+                and data_labeler_options.data_labeler_object is None:
+
+            try:
+
+                data_labeler = DataLabeler(
+                    labeler_type='unstructured',
+                    dirpath=data_labeler_options.data_labeler_dirpath,
+                    load_options=None)
+                self.options.set(
+                    {'data_labeler.data_labeler_object': data_labeler})
+
+            except Exception as e:
+                utils.warn_on_profile('data_labeler', e)
+                self.options.set({'data_labeler.is_enabled': False})
+
+        if data is not None:
+            self.update_profile(data)
+
+    def __add__(self, other):
+        """
+        Merges two profiles together overriding the `+` operator.
+
+        :param other: unstructured profile being add to this one.
+        :type other: UnstructuredProfiler
+        :return: merger of the two profiles
+        """
+        if type(other) is not type(self):
+            raise TypeError('`{}` and `{}` are not of the same profiler type.'.
+                            format(type(self).__name__, type(other).__name__))
+
+        merged_profile = UnstructuredProfiler(
+            data=None, samples_per_update=self._samples_per_update,
+            min_true_samples=self._min_true_samples, options=self.options
+        )
+        merged_profile.encoding = self.encoding \
+            if self.encoding == other.encoding else 'multiple files'
+        merged_profile.file_type = self.file_type \
+            if self.file_type == other.file_type else 'multiple files'
+        merged_profile._empty_line_count = (
+            self._empty_line_count + other._empty_line_count)
+        merged_profile.total_samples = self.total_samples + other.total_samples
+
+        samples = list(dict.fromkeys(self.sample + other.sample))
+        merged_profile.sample = random.sample(list(samples),
+                                              min(len(samples), 5))
+        merged_profile._profile = self._profile + other._profile
+        return merged_profile
+
+    def _get_sample_size(self, data):
+        """
+        Determines the minimum sampling size for detecting column type.
```

Comment:
Should this be updated? Theres no columns here its just a single column, right?

Reply:
fixed